driver-core/src/test/unit/com/mongodb/connection/DefaultServerConnectionSpecification.groovy

```diff
@@ -129,7 +129,7 @@ class DefaultServerConnectionSpecification extends Specification {
         def updates = asList(new UpdateRequest(new BsonDocument(), new BsonDocument(), WriteRequest.Type.REPLACE))
         def executor = Mock(ProtocolExecutor) {
             1 * execute({
-                compare(new UpdateCommandProtocol(namespace, true, UNACKNOWLEDGED, null, updates), it) },
+                compare(new UpdateCommandProtocol(namespace, true, ACKNOWLEDGED, null, updates), it) },
```

Comment:
Amazingly these were the only other code changes that were required after fixing CustomMatcher